src/videotile/VideoTileState.ts

```diff
@@ -108,6 +110,11 @@ export default class VideoTileState {
    */
   streamId: number | null = null;
 
+  /**
+   * Simulcast layer being received
+   */
+  simulcastLayers: SimulcastLayers = SimulcastLayers.High;
```

Comment:
It is from the design, @alochbau-amzn if we are not currently changing it yet, can we drop this, (like similarly we dropped it for `pausedDueToBandwidth` status)

Reply:
It is initialized to high since that will be the perceived configuration for the non-simulcast case.  For simulcast it will be updated with the actual values as they are determined